api/db-cdr/generate-cdr/run-measurement-queries.sh

```diff
@@ -994,6 +994,67 @@ and m1.measurement_source_concept_id > 0 and
 m1.measurement_source_concept_id not in (select distinct measurement_concept_id from \`${BQ_PROJECT}.${BQ_DATASET}.measurement\`)
 group by stratum_1,stratum_3,stratum_4,stratum_5"
 
+# Generating biological sex counts for measurement concepts for each unit
```

Comment:
I tried to stare at this for a while but I'm not really able to review it. Could you please expand this comment? I'm not really even sure what "unit" means in this context. Is this saying group by sex, then group by different measurement unit concept IDs, then count?

In the future, we should probably consider how we can try and refactor this file so it's more maintainable.

Reply:
yeah, Did not know of other way to it. Just building the queries based on previous ones. 
Measurement has different units (beats/ min, / sec and such) right, this query gets the biological sex count of each measurement for each unit. Eg: Heart rate can have 34000 male participants in which only 4000 could have their heart rate recorded in unit '/sec'.
This ftches the indvidual counts.

Reply:
Please add this comment to the code